aeppl/mixture.py

```diff
@@ -371,3 +434,14 @@ def logprob_MixtureRV(
     "basic",
     "mixture",
 )
+
+
+logprob_rewrites_db.register(
+    "switch_mixture_replace",
+    EquilibriumOptimizer(
+        [switch_mixture_replace], max_use_ratio=aesara.config.optdb__max_use_ratio
+    ),
+    0,
+    "basic",
+    "mixture",
+)
```

Comment:
These lines follow the registration of `mixture_replace` in `logprob_rewrites_db` very closely.